src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/SentenceCalcType.java

```diff
@@ -64,4 +66,13 @@ public static class PK implements Serializable {
     @Convert(converter = YesNoConverter.class)
     private boolean active;
 
+    public Boolean isAFine() {
+        return Objects.equals(calculationType, "A/FINE");
+    }
+
+    public Boolean isRecallType(){
+        var licenceRecallTypes = new String[]{"LR", "LR_ORA", "LR_YOI_ORA", "LR_SEC91_ORA", "LRSEC250_ORA"};
+        return Arrays.stream(licenceRecallTypes).filter(Objects::nonNull).anyMatch(licenceRecallType -> licenceRecallType.equals(calculationType));
```

Comment:
seems quite a convoluted way to find out if it a recall type, especially since `Objects::nonNull` will always be true since you've only just created the list on the previous line.  Would be better to just put the items into a `Set` instead and then you can do a `contains` rather than looping over all the values.

Would be also better to make the values into a constant called `RECALL_TYPES` or something.

Reply:
I've implemented this